docs/siem/detections/api/lists/api-delete-list-container.asciidoc

```diff
@@ -0,0 +1,29 @@
+[[lists-api-delete-container]]
+=== Delete list container
+
+Deletes a list container.
+
+==== Request URL
+
+`DELETE <kibana host>:<port>/api/lists`
+
+===== URL query parameters
+
+The URL query must include the list container's `id`:
+
+`id` - `DELETE /api/lists?id=<id>`
+
+===== Example request
+
+Deletes the list container with an `id` of `external-ip-excludes`:
+
+[source,console]
+--------------------------------------------------
+DELETE api/lists?id=external-ip-excludes
+--------------------------------------------------
+// KIBANA
+
+==== Response code
+
+`200`:: 
+    Indicates a successful call.
```

Comment:
You might want to mention that if you delete a container then it will in turn delete all of the container's items as well.